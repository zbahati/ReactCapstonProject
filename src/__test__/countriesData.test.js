import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/store';
import Main from '../components/main/main';

test('Main components matches snapshot', () => {
  const { container } = render(
    <Provider store={store}>
      <Router>
        <Main />
      </Router>
    </Provider>,
  );
  expect(container).toMatchSnapshot();
});
