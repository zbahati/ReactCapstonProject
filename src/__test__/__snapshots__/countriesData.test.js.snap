// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main components matches snapshot 1`] = `
<div>
  <main>
    <div
      class="navbar"
    >
      <a
        href="/"
      >
        <i
          class="fa-solid fa-arrow-left"
          id="back-button"
        />
      </a>
      <h3>
        All Countries
      </h3>
      <div
        class="nav-right"
      >
        <a
          href="/underContruction"
        >
          <i
            class="fa-solid fa-microphone"
          />
        </a>
        <a
          href="/underContruction"
        >
          <i
            class="fa-solid fa-gear"
          />
        </a>
      </div>
    </div>
    <div
      class="form"
    >
      <input
        class="input"
        placeholder="Search Country"
        type="text"
        value=""
      />
      <small>
        Search by LowerCase letters
      </small>
    </div>
    <div
      class="main-content"
    >
      <div>
        Loading ..
      </div>
    </div>
  </main>
</div>
`;
